content/doc/developer/publishing/releasing-cd.adoc

```diff
@@ -71,6 +71,8 @@ IMPORTANT: These files may have been set up this way by the https://github.com/j
 
 === Configure Dependabot
 
+NOTE: If you have a `.github/renovate.json`, do not configure Dependabot. link:https://www.jenkins.io/blog/2023/09/20/renovate-bot-probe-blog[For more details]
```

Comment:
We avoid placing the www.jenkins.io part into the URL inside the site because it makes it more difficult to test the site from a local build.
```suggestion
NOTE: If you have a `.github/renovate.json`, do not configure Dependabot. link:/blog/2023/09/20/renovate-bot-probe-blog[For more details]
```